Browser_IDE/stylesheet.css

```diff
@@ -472,4 +541,73 @@ li.CodeMirror-hint-active {
   position: relative;
   flex: 1 1 auto;
   padding: var(--bs-modal-padding);
-}
\ No newline at end of file
+}
+
+/* Modal UI changes */
+
+#confirmationModal .sk-header, #projectConflictModal .sk-header {
+  padding: 1.5em 1em; 
+  display: flex; 
+  align-items: center; 
+}
+
+#confirmationModal .modal-body, #projectConflictModal .modal-body {
+  border-top: 1px solid white;
+  border-bottom: 1px solid white; 
+}
+
+#confirmationModal .sk-modal-footer, #projectConflictModal .sk-modal-footer {
+  padding: 2.5em 1em; 
+  border-top: 1px solid white;
+  display: flex; 
+  align-items: center; 
+  justify-content: flex-end; 
+}
+
+
+.btn {
+  font-size: 1rem; 
+  padding: 0.4em 0.8em; 
+  border-radius: 0.25rem; 
+  margin-right: 0.5rem; 
+  margin-top: 0.5rem; 
+  transition: background-color 0.3s, box-shadow 0.3s; 
+}
+
+.btn:last-child {
+  margin-right: 0;
+}
+
+
+.btn-secondary {
+  background-color: #6c757d; 
+  color: white; 
+}
+
+.btn-secondary:hover, .btn-secondary:focus {
+  background-color: #5a6268; 
+}
+
+
+.btn-success {
+  background-color: #28a745; 
+  color: white; 
+}
+
+.btn-success:hover, .btn-success:focus {
+  background-color: #218838; 
+}
+
+/* Canvas scaling fix */
+#ExecutionEnvironment canvas {
```

Comment:
This has the right idea, but doesn't work currently:
![scalingtest](https://github.com/thoth-tech/SplashkitOnline/assets/42032199/b5a11cd2-2637-41f9-aa9b-c49e6710fc98)
 The fix is pretty easy - start by making sure it's actually applying to the canvas (the #ExecutionEnvironment won't be working), and then make sure the width and height are marked as important. You can test this in the browser using the Responsive Design/Viewport Mode - if it's fixed it'll look like this:
![scalingtestfixed](https://github.com/thoth-tech/SplashkitOnline/assets/42032199/12171f89-565c-4557-a960-94d25844eebf)